package.json

```diff
@@ -35,23 +35,22 @@
   "license": "MIT",
   "repository": "jest-community/jest-extended",
   "devDependencies": {
+    "@babel/cli": "^7.0.0",
```

Comment:
I'm hesitant to change the babel setup in this PR because it's unrelated. Can you revert and handle separately? @keeganwitt has been working on the babel upgrade in a separate stream of work.

Reply:
That's #327 for reference, which is blocked pending #328.

Reply:
As mentionned in the PR description, this bump is a direct dependency of the update of `babel-jest@>=24.0.0`, required because `jest@>=24.0.0` exposes context properties allowing specific behavior depending on the async context. I doubt this PR could be done reliabily and being compatible with `jest@^23.0.0` & `jest@^24.0.0`.